src/rules/javascript/complex_prefs_defaults.js

```diff
@@ -0,0 +1,26 @@
+import { COMPLEX_PREFS_DEFAULTS } from 'messages/javascript';
+
+export default function(context) {
+  var filename = context.getFilename();
+
+  if (filename.indexOf('defaults/preferences/') === 0) {
+    return {
+      CallExpression: function(node) {
+        var callee = node.callee;
```

Comment:
This seems a bit superfluous; doesn't help with readability much and doesn't shorten any lines; you can probably just use `node.callee` directly below.